<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>web-to-webm v.0.6.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/Resources.qrc">
    <normaloff>:/icons/yt-icon.png</normaloff>:/icons/yt-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QLineEdit" name="urlEdit">
         <property name="placeholderText">
          <string>Url to Youtube video</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="titleEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Youtube video title</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWebView" name="player">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">body{background:#333;}</string>
         </property>
         <property name="url">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>12</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="cutFromEdit">
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="placeholderText">
                <string>Cut from</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="cutToEdit">
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="placeholderText">
                <string>Cut to</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="filenameEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>Filename</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="selectSavePath">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/Resources.qrc">
                 <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QComboBox" name="qualityComboBox">
               <property name="editable">
                <bool>false</bool>
               </property>
               <property name="currentIndex">
                <number>-1</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="startConversion">
               <property name="minimumSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/Resources.qrc">
                 <normaloff>:/icons/convert.png</normaloff>:/icons/convert.png</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Downloading</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="downloadProgressBar">
            <property name="value">
             <number>0</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Conversion</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="conversionProgressBar">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="textVisible">
             <bool>true</bool>
            </property>
            <property name="textDirection">
             <enum>QProgressBar::TopToBottom</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="logBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="tabChangesFocus">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="centerOnScroll">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="menuDontConvert"/>
    <addaction name="menuRemoveAudio"/>
    <addaction name="menuRemoveRawVideo"/>
    <addaction name="menuDefaultDownloadPath"/>
    <addaction name="separator"/>
    <addaction name="menuYoutubedlCustomParams"/>
    <addaction name="menuFfmpegCustomParams"/>
    <addaction name="separator"/>
    <addaction name="menuCustomYoutubedlPath"/>
    <addaction name="menuCustomFfmpegPath"/>
    <addaction name="separator"/>
    <addaction name="menuShowFile"/>
    <addaction name="separator"/>
    <addaction name="menuResetAllSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="menuAbout"/>
    <addaction name="menuWebsite"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menuNew"/>
    <addaction name="separator"/>
    <addaction name="menuExit"/>
   </widget>
   <widget class="QMenu" name="menuLog">
    <property name="title">
     <string>Log</string>
    </property>
    <addaction name="menuYoutubedlOutput"/>
    <addaction name="menuFfmpegOutput"/>
    <addaction name="separator"/>
    <addaction name="menuShowLog"/>
    <addaction name="menuClearLog"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuLog"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="menuRemoveAudio">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remove audio</string>
   </property>
  </action>
  <action name="menuDefaultDownloadPath">
   <property name="text">
    <string>Set default download location</string>
   </property>
  </action>
  <action name="menuAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="menuRemoveRawVideo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remove raw video</string>
   </property>
  </action>
  <action name="menuFfmpegCustomParams">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add custom parameters for ffmpeg</string>
   </property>
  </action>
  <action name="menuYoutubedlCustomParams">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add custom parameters for youtube-dl</string>
   </property>
  </action>
  <action name="menuCustomYoutubedlPath">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set custom youtube-dl path</string>
   </property>
  </action>
  <action name="menuCustomFfmpegPath">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set custom ffmpeg path</string>
   </property>
  </action>
  <action name="menuWebsite">
   <property name="text">
    <string>Website</string>
   </property>
  </action>
  <action name="menuDontConvert">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Don't convert (download only)</string>
   </property>
  </action>
  <action name="menuShowLog">
   <property name="text">
    <string>Show full log</string>
   </property>
  </action>
  <action name="menuClearLog">
   <property name="text">
    <string>Clear log</string>
   </property>
  </action>
  <action name="menuNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="menuExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="menuShowFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open output directory after conversion</string>
   </property>
  </action>
  <action name="menuYoutubedlOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display youtube-dl output</string>
   </property>
  </action>
  <action name="menuFfmpegOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display ffmpeg output</string>
   </property>
  </action>
  <action name="menuResetAllSettings">
   <property name="text">
    <string>Reset all settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
